client/common/file.c

```diff
@@ -1614,38 +1614,48 @@ BOOL freerdp_client_populate_settings_from_rdp_file(rdpFile* file, rdpSettings*
 	{
 		if (file->BandwidthAutoDetect != 0)
 		{
+			if (file->NetworkAutoDetect != 0)
```

Comment:
This check is ambiguous as it is also true if NetworkAutoDetect is not set in the file. We should only output a warning if it is explicitly wrong in the rdp file.